packages/core/core-flows/src/draft-order/workflows/remove-draft-order-shipping-method.ts

```diff
@@ -83,12 +82,6 @@ export const removeDraftOrderShippingMethodWorkflow = createWorkflow(
 
     validateDraftOrderChangeStep({ order, orderChange })
 
-    updateOrderTaxLinesWorkflow.runAsStep({
```

Comment:
thought: should we include a test? 🙏

Reply:
yes, writing one now, this is why I left it as a draft PR